content/en/docs/refguide/modeling/application-logic/workflows/workflow-using-boundary-events.md

```diff
@@ -0,0 +1,77 @@
+
+- Windows: Preferences -> New Features -> Enable non-interrupting timer boundary events (beta)
+- Mac (Beta): Preferences -> New Features -> Enable non-interrupting timer boundary events (beta)
+
+{{< figure src="/attachments/refguide/modeling/application-logic/workflows/workflow-how-to-boundary-events/boundary-events-experimental-option.png" alt="Experimental option" width="450" class="no-border">}}
+
+## How to add a Timer Boundary Event
+A boundary event can be added to the following activities:
+
+- User Task
+- Multi-User Task
+- Wait for Notification
+- Call Workflow (aka sub-workflow)
+- Call Microflow
+
+To add a boundary event to one of these activities there are 2 ways:
+
+1. Select an Event activity (currently only Timer event is available) from the workflow Toolbox and drag it on one of the activities listed above. Double click the Timer Boundary Event icon to open the Timer properties dialog.
+
+{{< figure src="/attachments/refguide/modeling/application-logic/workflows/workflow-how-to-boundary-events/drag-drop.png" alt="Adding boundary events through drag and drop" width="200" class="no-border">}}
+
+2. Open the activity (e.g. User Task) properties (1), go to the Events tab (2) and create a new boundary event (3). Select the Timer event (4).
+
+{{< figure src="/attachments/refguide/modeling/application-logic/workflows/workflow-how-to-boundary-events/edit-dialog.png" alt="Adding boundary events through edit dialog" width="450" class="no-border">}}
+
+The next step is to configure the boundary timer properties like Caption and timer Duration. This could be a fixed period (after the parent task is initiated) or a date+time can be defined in an expression. For more information see [Timer Event](/refguide/timer-event/).
+
+## Execution
+
+Boundary events are initiated when their parent activity is initiated. For example, for a timer with a fixed duration, it will start its count down when the parent activity is initiated. In case the parent activity is completed before any of the boundary events are triggered, none of the activities in the boundary event path will be executed and all timers will be cancelled.
+
+WWith non-interrupting boundary events the parent activity remains active/in progress when an event is triggered (i.e., the parent activity is not interrupted). For example, when a timer event on a user task is triggered after 2 days, this task will remain in progress and the path defined below the timer event is executed. When the boundary event path reaches the **End of Boundary Path** activity, the workflow will await the completion of the parent activity. 
+
+## Current Limitations
+
+The current release (10.15) of boundary events has the following functional limitations (which are actively being developed):
+
+- No migration possible of in-progress workflows when boundary event paths change between deployments.
```

Comment:
Is this correct? I thought we support migrations in 10.15.

Reply:
You are correct, I removed this line.